spec/features/course/assessment_management_spec.rb

```diff
@@ -66,15 +66,17 @@
       end
     end
 
-    context 'As a Course Student' do
+    context 'As a Course Student', focus: true do
       let(:user) { create(:course_user, :approved, course: course).user }
       scenario 'I can see assessments' do
         category = course.assessment_categories.first
-        assessment = create(:assessment, course: course, tab: category.tabs.first)
+        assessment = create(:assessment, course: course, tab: category.tabs.first,
+                                         start_at: 1.day.from_now, end_at: 2.days.from_now)
         visit course_assessments_path(course)
 
         find_link(assessment.title, href: course_assessment_path(course, assessment)).click
         expect(current_path).to eq(course_assessment_path(course, assessment))
+        expect(Time.min..Time.max).to include(Time.now)
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.